src/HomeComponent/HomeComponent.js

```diff
@@ -4,62 +4,87 @@ import { Link } from 'react-router-dom';
 import { getObject } from '../exerciseStorageService';
 
 function HomeComponent () {
+  function weekStatus (n) {
+    const exerciseData = getObject();
+    const weekOneDayOneIsFinished = !!(exerciseData.finished.find(x => x[0] == n && x[1] == 1));
+    const weekOneDayTwoIsFinished = !!(exerciseData.finished.find(x => x[0] == n && x[1] == 2));
+    const weekOneDayThreeIsFinished = !!(exerciseData.finished.find(x => x[0] == n && x[1] == 3));
+
+    let weekOneStatus = 0;
```

Comment:
you can remove this weekOneStatus declaration and do the following like this:
  if (weekOneDayOneIsFinished) {
    return 1;
    }

You can create a constants.js file, and declare DAY_ONE=1, DAY_TWO=2 and DAY_3=3 and then refer them here, since to a new developer this may not be so easy to read. And overall this is a better practice to follow.

So this would look like:

  if (weekOneDayOneIsFinished) {
    return DAY_ONE;
    }